### PR TITLE
Add v0.3.0 changelog

### PR DESCRIPTION
**Pull request by hhaAndroid**
Thanks for your contribution and we appreciate it a lot. The following instructions would make your pull request more healthy and more easily get feedback. If you do not understand some items, don't worry, just make the pull request and seek help from maintainers.

## Motivation

Please describe the motivation for this PR and the goal you want to achieve through this PR.

## Modification

Please briefly describe what modification is made in this PR.

## BC-breaking (Optional)

Does the modification introduce changes that break the backward compatibility of the downstream repos?
If so, please describe how it breaks the compatibility and how the downstream projects should modify their code to keep compatibility with this PR.

## Use cases (Optional)

If this PR introduces a new feature, it is better to list some use cases here and update the documentation.

## Checklist

1. Pre-commit or other linting tools are used to fix potential lint issues.
2. The modification is covered by complete unit tests. If not, please add more unit tests to ensure the correctness.
3. If the modification has a potential influence on downstream projects, this PR should be tested with downstream projects, like MMDetection or MMClassification.
4. The documentation has been modified accordingly, like docstring or example tutorials.

### CODE REVIEW

File: docs/en/notes/changelog.md [hunk 1]
```diff
@@ -1,5 +1,69 @@
...
+1. Implement fast version of [RTMDet](https://github.com/open-mmlab/mmyolo/blob/dev/configs/rtmdet/README.md). RTMDet-s 8xA100 training takes only 14 hours.
+2. Support [PPYOLOE](https://github.com/open-mmlab/mmyolo/blob/dev/configs/ppyoloe/README.md) training
+3. Support `iscrowd` attribute training in [YOLOv5](https://github.com/open-mmlab/mmyolo/blob/dev/configs/yolov5/crowdhuman/yolov5_s-v61_8xb16-300e_ignore_crowdhuman.py)
+4. Support [YOLOv5 assigner result visualization](https://github.com/open-mmlab/mmyolo/blob/dev/projects/assigner_visualization/README.md)
+
+### New Features
+
+01. Add `crowdhuman` dataset (#368)
+02. Support PyTorch model forward for TensorRT inference (#377)
+03. Add `yolox` structure description (#402)
```
Comment (PeterH0323) [inline docs/en/notes/changelog.md:16]:
```suggestion
03. Add `YOLOX` structure description (#402)
```

File: docs/en/notes/changelog.md [hunk 2]
```diff
@@ -1,5 +1,69 @@
...
+12. Translate `rtmdet_description.md` document to English (#353)
+13. Add doc of `yolov6_description.md` (#382, #372)
+
+### Bug Fixes
+
+01. Fix bugs in the output annotation file when `--class-id-txt` is set (#430)
+02. Fix batch inference bug in `YOLOv5` head (#413)
+03. Fix typehint in some heads (#415, #416, #443)
+04. Fix RuntimeError of torch.cat() expected a non-empty list of Tensors (#376)
+05. Fix the device inconsistency error in `yolov7` training (#397)
```
Comment (PeterH0323) [inline docs/en/notes/changelog.md:34]:
```suggestion
05. Fix the device inconsistency error in `YOLOv7` training (#397)
```

File: docs/en/notes/changelog.md [hunk 3]
```diff
@@ -1,5 +1,69 @@
...
+11. Add docs about how to set the random seed (#386)
+12. Translate `rtmdet_description.md` document to English (#353)
+13. Add doc of `yolov6_description.md` (#382, #372)
+
+### Bug Fixes
+
+01. Fix bugs in the output annotation file when `--class-id-txt` is set (#430)
+02. Fix batch inference bug in `YOLOv5` head (#413)
+03. Fix typehint in some heads (#415, #416, #443)
+04. Fix RuntimeError of torch.cat() expected a non-empty list of Tensors (#376)
```
Comment (PeterH0323) [inline docs/en/notes/changelog.md:33]:
```suggestion
04. Fix RuntimeError of `torch.cat()` expected a non-empty list of Tensors (#376)
```

File: docs/en/notes/changelog.md [hunk 4]
```diff
@@ -1,5 +1,69 @@
...
+05. Fix the device inconsistency error in `yolov7` training (#397)
+06. Fix the `scale_factor` and `pad_param` value in `LetterResize` (#387)
+07. Fix docstring graph rendering error of readthedocs (#400)
+08. Fix AssertionError when `YOLOv6` from training to val (#378)
+09. Fix CI error due to `np.int` and legacy builder.py (#389)
+10. Fix mmdeploy rewriter (#366)
+11. Fix mmyolo unittest scope bug (#351)
```
Comment (PeterH0323) [inline docs/en/notes/changelog.md:40]:
```suggestion
10. Fix MMDeploy rewriter (#366)
11. Fix MMYOLO unittest scope bug (#351)
```

File: docs/zh_cn/notes/changelog.md [hunk 5]
```diff
@@ -1,5 +1,74 @@
...
+1. 实现了 [RTMDet](https://github.com/open-mmlab/mmyolo/blob/dev/configs/rtmdet/README.md) 的快速版本。RTMDet-s 8xA100 训练只需要 14 个小时。
+2. 支持 [PPYOLOE](https://github.com/open-mmlab/mmyolo/blob/dev/configs/ppyoloe/README.md) 训练。
+3. 支持 [YOLOv5](https://github.com/open-mmlab/mmyolo/blob/dev/configs/yolov5/crowdhuman/yolov5_s-v61_8xb16-300e_ignore_crowdhuman.py) 的 `iscrowd` 属性训练。
+4. 支持 [YOLOv5 正样本分配结果可视化](https://github.com/open-mmlab/mmyolo/blob/dev/projects/assigner_visualization/README.md)
+
+### 新特性
+
+01. 新增 `crowdhuman` 数据集 (#368)
+02. EasyDeploy 中支持 PyTorch 模型的 TensorRT 推理的 (#377)
+03. 新增 `yolox` 结构图描述 (#402)
```
Comment (PeterH0323) [inline docs/zh_cn/notes/changelog.md:16]:
```suggestion
03. 新增 `YOLOX` 结构图描述 (#402)
```

File: docs/zh_cn/notes/changelog.md [hunk 6]
```diff
@@ -1,5 +1,74 @@
...
+12. 将 `rtmdet_description.md` 文档翻译成英文 (#353)
+13. 新增 `yolov6_description.md` 文档 (#382, #372)
+
+### Bug 修复
+
+01. 修复设置 `--class-id-txt` 时输出注释文件中的错误 (#430)
+02. 修复 `YOLOv5` head 中的批量推理错误 (#413)
+03. 修复某些 head 的类型提示（#415、#416、#443）
+04. 修复 expected a non-empty list of Tensors (#376)
+05. 修复 `yolov7` 训练中的设备不一致错误（#397）
```
Comment (PeterH0323) [inline docs/zh_cn/notes/changelog.md:34]:
```suggestion
05. 修复 `YOLOv7` 训练中的设备不一致错误（#397）
```

File: docs/zh_cn/notes/changelog.md [hunk 7]
```diff
@@ -1,5 +1,74 @@
...
+06. 修复 `LetterResize` 中的 `scale_factor` 和 `pad_param` 值 (#387)
+07. 修复 readthedocs 的 docstring 图形渲染错误 (#400)
+08. 修复 `YOLOv6` 从训练到验证时的断言错误 (#378)
+09. 修复 `np.int` 和旧版 builder.py 导致的 CI 错误 (#389)
+10. 修复 mmdeploy 重写器 (#366)
+11. 修复 mmyolo 单元测试错误 (#351)
```
Comment (PeterH0323) [inline docs/zh_cn/notes/changelog.md:40]:
```suggestion
10. 修复 MMDeploy 重写器 (#366)
11. 修复 MMYOLO 单元测试错误 (#351)
```

File: docs/zh_cn/notes/changelog.md [hunk 8]
```diff
@@ -1,5 +1,74 @@
...
+
+### 完善
+
+01. 更新 `useful_tools.md` (#384)
+02. 更新英文版 `custom_dataset.md` (#381)
+03. 从重写函数中删除上下文参数 (#395)
+04. 弃用 `np.bool` 类型别名 (#396)
+05. 为自定义数据集添加新的视频链接 (#365)
+06. 仅为模型导出 onnx (#361)
+07. 添加 mmyolo 回归测试 yml (#359)
```
Comment (PeterH0323) [inline docs/zh_cn/notes/changelog.md:53]:
```suggestion
07. 添加 MMYOLO 回归测试 yml (#359)
```

File: docs/en/notes/changelog.md [hunk 9]
```diff
@@ -1,5 +1,69 @@
...
+
+### Improvements
+
+01. Update `useful_tools.md` (#384)
+02. update the English version of `custom_dataset.md` (#381)
+03. Remove context argument from the rewriter function (#395)
+04. deprecating `np.bool` type alias (#396)
+05. Add new video link for custom dataset (#365)
+06. Export onnx for model only (#361)
+07. Add mmyolo regression test yml (#359)
```
Comment (PeterH0323) [inline docs/en/notes/changelog.md:53]:
```suggestion
07. Add MMYOLO regression test yml (#359)
```

File: docs/zh_cn/notes/changelog.md [hunk 10]
```diff
@@ -1,5 +1,74 @@
...
+
+1. 实现了 [RTMDet](https://github.com/open-mmlab/mmyolo/blob/dev/configs/rtmdet/README.md) 的快速版本。RTMDet-s 8xA100 训练只需要 14 个小时。
+2. 支持 [PPYOLOE](https://github.com/open-mmlab/mmyolo/blob/dev/configs/ppyoloe/README.md) 训练。
+3. 支持 [YOLOv5](https://github.com/open-mmlab/mmyolo/blob/dev/configs/yolov5/crowdhuman/yolov5_s-v61_8xb16-300e_ignore_crowdhuman.py) 的 `iscrowd` 属性训练。
+4. 支持 [YOLOv5 正样本分配结果可视化](https://github.com/open-mmlab/mmyolo/blob/dev/projects/assigner_visualization/README.md)
+
+### 新特性
+
+01. 新增 `crowdhuman` 数据集 (#368)
+02. EasyDeploy 中支持 PyTorch 模型的 TensorRT 推理的 (#377)
```
Comment (PeterH0323) [inline docs/zh_cn/notes/changelog.md:15]:
```suggestion
02. EasyDeploy 中支持 TensorRT 推理 (#377)
```

File: docs/en/notes/changelog.md [hunk 11]
```diff
@@ -1,5 +1,69 @@
 # Changelog
 
+## v0.3.0 (8/1/2023)
+
+### Highlights
+
+1. Implement fast version of [RTMDet](https://github.com/open-mmlab/mmyolo/blob/dev/configs/rtmdet/README.md). RTMDet-s 8xA100 training takes only 14 hours.
```
Comment (RangeKing) [inline docs/en/notes/changelog.md:7]:
```suggestion
1. Implement fast version of [RTMDet](https://github.com/open-mmlab/mmyolo/blob/dev/configs/rtmdet/README.md). RTMDet-s 8xA100 training takes only 14 hours. The training speed is 2.6 times faster than the previous version.
```

File: docs/zh_cn/notes/changelog.md [hunk 12]
```diff
@@ -1,5 +1,74 @@
 # 更新日志
 
+## v0.3.0 (8/1/2023)
+
+### 亮点
+
+1. 实现了 [RTMDet](https://github.com/open-mmlab/mmyolo/blob/dev/configs/rtmdet/README.md) 的快速版本。RTMDet-s 8xA100 训练只需要 14 个小时。
```
Comment (RangeKing) [inline docs/zh_cn/notes/changelog.md:7]:
```suggestion
1. 实现了 [RTMDet](https://github.com/open-mmlab/mmyolo/blob/dev/configs/rtmdet/README.md) 的快速版本。RTMDet-s 8xA100 训练只需要 14 个小时。训练速度比之前的版本快 2.6 倍。
```

File: docs/zh_cn/advanced_guides/index.rst [hunk 13]
```diff
@@ -1,3 +1,11 @@
+模块组合
+************************
+
+.. toctree::
+   :maxdepth: 1
+
+   module_combination.md
+
```
Comment (RangeKing) [inline docs/zh_cn/advanced_guides/index.rst:8]: `advanced_guides/index.rst` in `docs/en` also need to be modified.

File: docs/en/notes/changelog.md [hunk 14]
```diff
@@ -1,5 +1,69 @@
...
+2. Support [PPYOLOE](https://github.com/open-mmlab/mmyolo/blob/dev/configs/ppyoloe/README.md) training
+3. Support `iscrowd` attribute training in [YOLOv5](https://github.com/open-mmlab/mmyolo/blob/dev/configs/yolov5/crowdhuman/yolov5_s-v61_8xb16-300e_ignore_crowdhuman.py)
+4. Support [YOLOv5 assigner result visualization](https://github.com/open-mmlab/mmyolo/blob/dev/projects/assigner_visualization/README.md)
+
+### New Features
+
+01. Add `crowdhuman` dataset (#368)
+02. Support PyTorch model forward for TensorRT inference (#377)
```
Comment (PeterH0323) [inline docs/en/notes/changelog.md:15]:
```suggestion
02. Easydeploy support TensorRT inference (#377)
```

**Pull request merged**